adapter/circom/go.mod

```diff
@@ -0,0 +1,16 @@
+module github.com/iden3/go-merkletree-sql/adapter/circom
+
+go 1.19
```

Comment:
Please switch here and in other places to 1.18. Let's follow the best practice to support the last 2 major releases. In our case it's 1.18 & 1.19.